Update Venafi Setup Guide (#4557)
pulumi/docs: content/docs/intro/cloud-providers/venafi/setup.md

Once obtained, there are two ways to communicate your authorization tokens to Pulumi when talking to Venafi TPP:

> 1. Set the environment variables `VENAFI_URL`, `VENAFI_TOKEN` and `VENAFI_ZONE`:

Reviewer: ```suggestion
1. By setting the environment variables `VENAFI_URL`, `VENAFI_TOKEN` and `VENAFI_ZONE`:
```

> 1. By setting the environment variables `VENAFI_URL`, `VENAFI_TOKEN` and `VENAFI_ZONE`:

Reviewer: ```suggestion
1. Setting the environment variables `VENAFI_URL`, `VENAFI_TOKEN` and `VENAFI_ZONE`:
```

    ```bash
    $ export VENAFI_URL=XXXXXXXXXXXXXX
    $ export VENAFI_TOKEN=YYYYYYYYYYYYYY
    $ export VENAFI_ZONE=ZZZZZZZZZZZZZZ
    ```

> 2. Set them using configuration, if you prefer that they be stored alongside your Pulumi stack for easy multi-user access:

Reviewer: ```suggestion
2. Using configuration, if you prefer storing authorization tokens alongside your Pulumi stack for easy multi-user access:
```

    ```bash
    $ pulumi config set venafi:url XXXXXXXXXXXXXX
    $ pulumi config set venafi:accessToken YYYYYYYYYYYYYY --secret
    $ pulumi config set venafi:zone ZZZZZZZZZZZZZZ
    ```

Remember to pass `--secret` when setting `venafi:accessToken` so that it is properly encrypted.

##### Generating an Access Token

> To generate an access token for TPP Admin, then we can make a `POST` request to the TPP Admin URL. An example would be as

Reviewer: ```suggestion
To generate an access token for TPP Admin, make a `POST` request to the TPP Admin URL. An example would be as
```
